core/src/main/scala/org/apache/spark/storage/BlockManagerManagedBuffer.scala

```diff
@@ -31,17 +35,31 @@ import org.apache.spark.util.io.ChunkedByteBuffer
 private[storage] class BlockManagerManagedBuffer(
     blockInfoManager: BlockInfoManager,
     blockId: BlockId,
-    chunkedBuffer: ChunkedByteBuffer) extends NettyManagedBuffer(chunkedBuffer.toNetty) {
+    data: BlockData,
+    dispose: Boolean) extends ManagedBuffer {
```

Comment:
Hmm, I prefer `dispose`, because it's not about needing to dispose the buffer, but wanting to dispose the buffer.